.github/workflows/dlp-pipelines.yml

```diff
@@ -214,9 +230,29 @@ jobs:
             else
               sleep 30s
             fi
+            echo "Got number of rows in ${{env.INPUT_FILE_NAME}}: $row_count."
           done
           echo "# records in input CSV file are: $rc_orig."
           echo "Verified number of rows in ${{env.INPUT_FILE_NAME}}: $row_count."
+      - name: Verify distinct rows of newly added file
```

Comment:
Please add a new line before starting a new step. We should follow same convention throughout the workflow.

Reply:
Will do and resolve. Thanks.